### PR TITLE
Fix #1156 : add submission guidelines

### LINKED ISSUE #1156: Show challenge 'submission guidelines' on the evaluation page of Challenge
## Observed Behaviour

Currently, we are storing the `submission guidelines` field for a challenge in the database but we are not displaying on the challenge page.

## Objective

The objective is to show the `submission guidelines` on the Evaluation tab of a challenge on EvalAI.

### PR DESCRIPTION
**Pull request by the-dagger**
Fix : #1156 

screen shot 2017-08-11 at 12 20 02 pm

@deshraj Please review.

### CODE REVIEW

**Comment by deshraj** [comment]
@the-dagger can you please allow the submission guidelines to occupy the full width of the screen? It is looking odd as of now.

**Comment by the-dagger** [comment]
Hm, that would look odd as well.
Let me experiment a bit with card width and height and get back to you.

**Comment by the-dagger** [comment]
@deshraj How does it look now?
screen shot 2017-08-11 at 1 53 43 pm

**Comment by deshraj** [comment]
No, this doesn't look good. Please try this: 
> @the-dagger can you please allow the submission guidelines to occupy the full width of the screen? It is looking odd as of now.

Also, can you please make sure that the submission guidelines is shown in the first row and others in the second row?

**Comment by the-dagger** [comment]
Ok, on it.

**Comment by the-dagger** [comment]
@deshraj Done.
screen shot 2017-08-11 at 2 19 05 pm

File: .gitignore [hunk 1]
```diff
@@ -89,4 +89,4 @@ challenge_data/*
 media/
 
 # .DS_Store for OSX
-*/.DS_Store
+*.DS_Store
```
Comment (taranjeet) [inline .gitignore]: Please update this in other PR. :)

File: frontend/src/views/web/challenge/evaluation.html [hunk 2]
```diff
@@ -1,5 +1,12 @@
 <section class="ev-sm-container ev-view">
     <div class="row">
+      <div class="col m12">
+          <div class="ev-md-container ev-card-panel ev-z-depth-5 ev-challenge-view  eval-card">
```
Comment (taranjeet) [inline frontend/src/views/web/challenge/evaluation.html]: One extra space between `ev-challenge-view` and `eval-card`

File: frontend/src/views/web/challenge/evaluation.html [hunk 3]
```diff
@@ -1,5 +1,12 @@
...
+              <span class="fs-18"><strong>Submission Guidelines
```
Comment (taranjeet) [inline frontend/src/views/web/challenge/evaluation.html]: Also I think we are missing alignment here ?

**Comment by deshraj** [comment]
@the-dagger can you please update the PR?

**Comment by the-dagger** [comment]
@deshraj done.
Also updated the gitignore file in #1192

**Comment by aayusharora** [comment]
Looks good to me 👍  Thanks @the-dagger

File: .gitignore [hunk 4]
```diff
@@ -87,6 +87,3 @@ challenge_data/*
 
 # Media Files
 media/
-
-# .DS_Store for OSX
-*/.DS_Store
```
Comment (deshraj) [inline .gitignore]: Why are we removing the lines 90, 91 and 92 here? These should not be the part of this PR, right?

Comment (the-dagger) [inline .gitignore]: @deshraj Done, fixed it.

**Pull request merged**